docs/docker-backend-nemesis.md

```diff
@@ -0,0 +1,91 @@
+# Individual Nemesis cases status on Docker backend
+| Nemesis | Status | Comment
+| :------ | :----- | :------
+| AbortRepairMonkey | Pass |
+| AddDropColumnMonkey | Pass |
+| AddRemoveDcNemesis  | Skipped | Requires multi-dc env
```

Comment:
actually it does not require multi-dc env - did you try it?

Reply:
The case was tried and failing. This and the fact that the case has a skip for K8S backend, as not supporting the multi dc env, made me think that the disruption is not supported for containerized backend in SCT

This is the error that the case is failing with now on Docker backend. 
```
----- LAST ERROR EVENT -------------------------------------------------------
2024-04-18 09:06:52.191: (DisruptionEvent Severity.ERROR) period_type=end event_id=0f2b5ecb-1042-4d08-8ec6-db8aec3e2f0a duration=6m28s: nemesis_name=AddRemoveDc target_node=Node longevity-1gb-1h-nemesis-dmitriy-db-node-febaa291-2 [172.17.0.4 | 172.17.0.4] errors=new datacenter was not registered
Traceback (most recent call last):
File "/home/dmitriy/Work/Scylla/scylla-cluster-tests/sdcm/nemesis.py", line 5117, in wrapper
result = method(*args[1:], **kwargs)
File "/home/dmitriy/Work/Scylla/scylla-cluster-tests/sdcm/nemesis.py", line 4534, in disrupt_add_remove_dc
assert new_dc_list, "new datacenter was not registered"
AssertionError: new datacenter was not registered
```

I will create a task for fixing support of Docker backend for the disruption.

Reply:
Create d an issue for this https://github.com/scylladb/scylla-cluster-tests/issues/7351

Reply:
Updated the table with the latest status for AddRemoveDcNemesis.